containers/Uniswap.ts

```diff
@@ -27,6 +27,7 @@ function getPair() {
     if (!client) return;
     try {
       const info = findInfoByAddress(address);
+      setError(null);
```

Comment:
there was a bug that if your uniswap endpoint threw an error at any point, the error would never be cleared even if the call succeeded later.  It would require a reload to fix. This will clear the error every time before the request is made.